.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+### Introduction
+
+[KVIrc](https://github.com/kvirc/KVIrc) and related projects ([KVIrc website](https://github.com/kvirc/kvirc-www), [KVIrc themes](https://github.com/kvirc/kvirc-themes) and [KVIrc scripts](https://github.com/kvirc/kvirc-scripts)) now all use GitHub for development, i.e. for source maintenance, *pull requests* and the review of such code.   
+Not familiar with Git? Start by looking at GitHub's [collaborating pages](https://help.github.com/categories/collaborating/).
+
+The *pull request* process is important, especially the code submission part, KVIrc uses the automatic **travis** build system to compile the *pull request* and allows the person(s) involved the time to make any improvements or fix any issues arising before the **pr** is merged into the main source tree. 

```

Comment:
`The *pull request* process is important, especially the code submission part` whats the difference? Also you should add a . afterwords otherwise it becomes a run on sentence.